modular_skyrat/master_files/code/modules/mob/living/carbon/human.dm

```diff
@@ -0,0 +1,12 @@
+/mob/living/carbon/human/ZImpactDamage(turf/T, levels)
+	//Non cat-people smash into the ground
+	if(!isfelinid(src))
```

Comment:
```suggestion
	if(!(isfelinid(src) || HAS_TRAIT(H, TRAIT_FELINE)))
```
fuuuuuuu

Reply:
In my personal opinion, I got a feeling like it gives a good advantage to that quirk. How much points does it cost?

Reply:
yeah, if i'm doin' that, may wanna boost the quirk points

Reply:
It is currently free, so I wouldn't link this with the Feline trait.
It was supposed to just be for some flavor stuff.
(I honestly think it was kinda poorly designed in the first place, but besides the point.)

Reply:
> 
> 
> It is currently free, so I wouldn't link this with the Feline trait.
> It was supposed to just be for some flavor stuff.
> (I honestly think it was kinda poorly designed in the first place, but besides the point.)

correct as always
![catkiss](https://user-images.githubusercontent.com/58394696/131239172-f913edf1-8abf-4282-bd93-3adfbdbff13d.gif)